samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx

```diff
@@ -98,6 +100,20 @@ const GitHubProjectSelection: FC<IData> = ({ uri, keyConfig, prevProject, prevBr
                     placeholder="https://github.com/microsoft/semantic-kernel"
                 />
             </div>
+            <Label>
+                <strong>GitHub PAT Token (optional)</strong>
```

Comment:
nit on the wording, perhaps one of: _GitHub PAT (optional)_ or _GitHub Personal Access Token (optional)_. 

Might be worth adding an open in new window link to explain what the PAT is and how to get one, ie: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens

Reply:
Adding link. Thanks @adrianwyatt  for the rewording.